scripts/sparkclr-repl.cmd

```diff
@@ -0,0 +1,4 @@
+rem Copyright (c) Microsoft. All rights reserved.
+rem Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+%SPARKCLR_HOME%\scripts\sparkclr-submit.cmd --name MobiusShell %* --exe Repl.exe %SPARKCLR_HOME%\repl
```

Comment:
Just a crazy idea. Do we want to support Debug mode in REPL?

I would love to have that feature even with some limitations.